guides/v2.3/release-notes/bk-release-notes.md

```diff
@@ -21,7 +21,7 @@ The ece-tools package is a scalable deployment tool that simplifies the {{ sit
 
 See [Release Notes for ece-tools]{{ page.baseurl }}/cloud/release-not4es/cloud-tools.html) for the latest updates and improvements to the ece-tools package as well as information about Magento Commerce Cloud upgrades and patches.
 
-{: .bs-callout .bs-callout-info } We recommend installing full [{{site.data.var.ece}} upgrades]({{ site.baseurl }}/guides/v2.3/cloud/project/project-upgrade.html) for important security updates. Full upgrades include all associated patches and hotfixes.
+{: .bs-callout .bs-callout-info } We recommend installing full {{site.data.var.ece}} upgrades for important security updates. Full upgrades include all associated patches and hotfixes.
```

Comment:
Why is this obsolete? Did something happen?

Reply:
> Why is this obsolete? Did something happen?
I removed the link because it puts you in a sub-topic of the Upgrades and Patches content. Because this is the Release Information, it seemed clearer to just link to the Release Notes for ece-tools topic, which provides a link to the upgrades and patches topic in the intro.

Also, I tried to minimize the amount of linked info in this Release Information topic to make it easier to maintain.